jdi-light-applitools-tests/src/main/java/jdisite/sections/ContactForm.java

```diff
@@ -0,0 +1,25 @@
+package jdisite.sections;
+
+/**
+ * Created by Roman Iovlev on 10.11.2018
+ * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
+ */
+
+import com.epam.jdi.light.elements.complex.dropdown.DropdownSelect;
+import com.epam.jdi.light.elements.composite.Form;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.*;
+import com.epam.jdi.light.ui.html.elements.complex.DataListOptions;
+import jdisite.custom.MultiSelect;
+import jdisite.entities.ContactInfo;
+
+public class ContactForm extends Form<ContactInfo> {
+    TextField name, lastName, position, passportNumber, passportSeria;
+
+    DropdownSelect gender;
+    DataListOptions religion;
+    @UI("#weather") MultiSelect weather;
+
+    Checkbox passport, acceptConditions;
+    TextArea description;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5478709)